src/aggregator/aggregator/aggregator.go

```diff
@@ -235,6 +242,44 @@ func (agg *aggregator) AddForwarded(
 	return nil
 }
 
+func (agg *aggregator) AddPassthrough(
+	metric aggregated.Metric,
+	storagePolicy policy.StoragePolicy,
+) error {
+	callStart := agg.nowFn()
+	agg.metrics.passthrough.Inc(1)
+
+	if agg.electionManager.ElectionState() == FollowerState {
+		agg.metrics.addPassthrough.ReportFollowerNoop()
+		agg.metrics.addPassthrough.ReportSuccess(agg.nowFn().Sub(callStart))
```

Comment:
hm don't know if you should put this in for the follower, it'll skew your timings pretty heavily